<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0" columnstretch="0,0" rowminimumheight="0,0" columnminimumwidth="0,0">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="build_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Yep You Guessed It This Builds The Ouput</string>
       </property>
       <property name="text">
        <string>Build</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="folderLabel">
         <property name="text">
          <string>Destination</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLineEdit" name="input_folder_path">
           <property name="toolTip">
            <string>The Destination For The Output Image</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browse_Button">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>File Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="input_file_name">
         <property name="toolTip">
          <string>The Rended Image File Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="framePaddingLabel">
         <property name="text">
          <string>Frame Padding</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QSpinBox" name="input_frame_padding">
           <property name="toolTip">
            <string>The Number Of 0 Added To The File Name Before The Current Frame Number</string>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="prefix">
            <string/>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Digits</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="Layers_Tab">
        <attribute name="title">
         <string>Layer Order</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="Layer_Order_UI" name="Layers_Order_Widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Views_Tab">
        <attribute name="title">
         <string>View Output</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="Nuke_Views_Selector_UI" name="Nuke_Views_Selector" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>300</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Layer_Order_UI</class>
   <extends>QWidget</extends>
   <header>layer_order_ui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Nuke_Views_Selector_UI</class>
   <extends>QWidget</extends>
   <header>nuke_views_selector_ui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
